tests/Fixer/PhpUnit/PhpUnitMockFixerTest.php

```diff
@@ -151,11 +154,21 @@ public function testFoo()
     }
 
     /**
-     * @requires PHP 8.0
+     * @requires PHP >= 8.0
```

Comment:
```suggestion
     * @requires PHP 8.0
```

Reply:
Why? Iimplicit over explicit?

Reply:
I know your opinion on it from previous PR.
Same time, I do not want to apply this change for whole repository, and even more for subset of it.

Reply:
I know you "do not enjoy the massive changes in `@requires PHP` annotations".
I wonder if there is any other reason. Usually, there are no objections to massive changes if they are good, and making the constraint explicit over implicit is IMHO a good reason.

Reply:
> I do not want to apply this change

thanks